superset/assets/javascripts/SqlLab/actions.js

```diff
@@ -326,8 +327,31 @@ export function popStoredQuery(urlId) {
           autorun: newQuery.autorun ? newQuery.autorun : false,
           sql: newQuery.sql ? newQuery.sql : 'SELECT ...',
         };
+        notify.success('Query loaded in a new tab from link');
         dispatch(addQueryEditor(queryEditorProps));
       },
+      error: () => notify.error("The query couldn't be loaded"),
+    });
+  };
+}
+export function popSavedQuery(saveQueryId) {
+  return function (dispatch) {
+    $.ajax({
+      type: 'GET',
+      url: `/savedqueryviewapi/api/get/${saveQueryId}`,
+      success: (data) => {
+        const sq = data.result;
+        const queryEditorProps = {
+          title: sq.label,
+          dbId: sq.db_id,
+          schema: sq.schema,
+          autorun: false,
+          sql: sq.sql,
+        };
+        dispatch(addQueryEditor(queryEditorProps));
+        notify.success('Query loaded in a new tab from link');
```

Comment:
i don't think we need to add a success notification here since the user will see that this has happened in the ui. it's only on error that the notification is important here i think.